FinSearchUnified/FinSearchUnified.php

```diff
@@ -54,6 +56,29 @@ public function install(InstallContext $context)
     {
         $context->scheduleClearCache([InstallContext::CACHE_TAG_THEME]);
         parent::install($context);
+
+        $shopwareVersion = Shopware()->Config()->get('version');
+
+        if (version_compare($shopwareVersion, '5.2.17', '>=')) {
+            $file = $this->getPath() . '/Resources/config/config.xml';
+        } else {
+            // For Shopware <= 5.2.17 we load a different config file due to missing `button` element in configuration
+            $file = $this->getPath() . '/Resources/config/config_compat.xml';
```

Comment:
Why not just load one config and add the button via code? All options except the button are duplicates.